ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/entities/TreeRelation.java

```diff
@@ -1,38 +1,46 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.entities;
 
-import org.apache.jena.rdf.model.Literal;
-import org.apache.jena.rdf.model.Resource;
-import org.apache.jena.rdf.model.ResourceFactory;
+import java.util.Objects;
 
 public class TreeRelation {
-    private String treePath;
-    private String treeValue;
-    private String treeNode;
-    private String rdfSyntaxType;
+    private final String treePath;
+    private final String treeValue;
+    private final String treeNode;
+    private final String rdfSyntaxType;
```

Comment:
Did you test this? having this final and not having a default contructor gave an issue with MongoDB

Reply:
Reverted this. Didn't realize that this domain class was also used as part of the MongoDB entity.